docs/spec.md

```diff
@@ -6184,6 +6178,20 @@ on types" section via `member_name`.
 
 #### Quantization computations
 
+* `element_type` is defined on tensor types and quantized tensor types and
```

Comment:
Let's put this function in the list of functions in this section in alphabetical order.